opslevel/resource_opslevel_team.go

```diff
@@ -174,8 +174,8 @@ func resourceTeamCreate(d *schema.ResourceData, client *opslevel.Client) error {
 		ManagerEmail:     d.Get("manager_email").(string),
 		Responsibilities: d.Get("responsibilities").(string),
 	}
-	if group, ok := d.GetOk("group"); ok {
-		input.Group = opslevel.NewIdentifier(group.(string))
```

Comment:
`input.Group` is removed in opslevel-go so it is necessary to throw errors here too.